examples/floating_interdomain/usecases/FloatingVl3/cluster1/kustomization.yaml

```diff
@@ -7,9 +7,7 @@ namespace: ns-vl3-interdomain
 resources:
   - namespace.yaml
   - nse-vl3-vpp.yaml
-
-bases:
-- ../../../../../apps/nsc-kernel
+  - ../../../../../apps/nsc-kernel
```

Comment:
Why don't we use refs for these resources?

Reply:
In fact, we don’t have to have refs here - it’s enough to use only the kustomization file by ref, it will be able to pick up all internal resources (by relative links)